app/models/container_image_registry.rb

```diff
@@ -4,4 +4,10 @@ class ContainerImageRegistry < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
   has_many :containers, :through => :container_images
+
+  virtual_column :host_port, :type => :string
+
+  def host_port
+    port.nil? ? host : host + ":" + port
```

Comment:
@zeari please also rename this to `def full_name` or something like `canonical_name`